src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <h1 class="heading">Leaderboard</h1>
+  <div class="main-container">
```

Comment:
- [ ] [OPTIONAL] It would be nice to use section tags instead of a more generic div tag, this makes your page better  and easy to optimize for SEO.

Reply:
Hi @danmainah, thanks a lot for the feedback. I appreciate you pointing this issue out, I will fix it right away. 👍🏾